Lib/scilab/scirun.swg

```diff
@@ -58,29 +57,29 @@ static void SWIG_Scilab_SetFuncName(char *funcName) {
 
 /* Api context management functions */
 
-#if SWIG_SCILAB_VERSION >= 600
+#if SCI_VERSION_MAJOR < 6
+#define SWIG_Scilab_SetApiContext(apiCtx)
+#else
 static void *pvApiCtx = NULL;
 static void SWIG_Scilab_SetApiContext(void *apiCtx) {
   pvApiCtx = apiCtx;
 }
-#else
-#define SWIG_Scilab_SetApiContext(apiCtx)
 #endif
 
 /* Argument management functions */
 
-#if SWIG_SCILAB_VERSION >= 540
-#define SWIG_CheckInputArgument(pvApiCtx, minInputArgument, maxInputArgument) CheckInputArgument(pvApiCtx, minInputArgument, maxInputArgument)
-#define SWIG_CheckInputArgumentAtLeast(pvApiCtx, minInputArgument) CheckInputArgumentAtLeast(pvApiCtx, minInputArgument)
-#define SWIG_CheckOutputArgument(pvApiCtx, minOutputArgument, maxOutputArgument) CheckOutputArgument(pvApiCtx, minOutputArgument, maxOutputArgument)
-#define SWIG_NbInputArgument(pvApiCtx) nbInputArgument(pvApiCtx)
-#define SWIG_AssignOutputArgument(pvApiCtx, outputArgumentPos, argumentPos) AssignOutputVariable(pvApiCtx, outputArgumentPos) = argumentPos
-#else
+#if SCI_VERSION_MAJOR < 6
```

Comment:
This has changed the version number being compared against as well as how the check was done - is that intended?

Reply:
good catch, I will change this `#if` to match the same version as before.